### PR TITLE
Update Bluetooth adapters

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Now that I know what to look for when ordering these, I found
quite a few more of the better performing ones as the quality
of the Linux driver seems to be the most important factor.

Another pile of adapters arrived today so I'm updating
the docs again.

Document the slow startup of the Broadcom adapters
on the Odroid n2+

Document the lack of reset pin on the RTLs

The test box has grown to two test boxes:
IMG_0068

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bluetooth.markdown [hunk 1]
```diff
@@ -102,6 +104,19 @@ Performance testing used the following hardware:
...
 
+### Known working adapters with missing firmware patch files
+
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
+- Cable Matters 604002-BLK (BCM20702A1) - USB id 0a5c:21e8
+- Kinivo BTD-400 (BCM20702A1)
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- Kinivo BTD-400 (BCM20702A1) - USB id 0a5c:21e8
```

File: source/_integrations/bluetooth.markdown [hunk 2]
```diff
@@ -102,6 +104,19 @@ Performance testing used the following hardware:
...
 
+### Known working adapters with missing firmware patch files
+
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
+- Cable Matters 604002-BLK (BCM20702A1) - USB id 0a5c:21e8
+- Kinivo BTD-400 (BCM20702A1) - USB id 0a5c:21e8
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- Kinivo BTD-400 (BCM20702A0) - USB id 0a5c:21e8
```

File: source/_integrations/bluetooth.markdown [hunk 3]
```diff
@@ -102,6 +105,19 @@ Performance testing used the following hardware:
...
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
+- Cable Matters 604002-BLK (BCM20702A1) - USB id 0a5c:21e8
+- Kinivo BTD-400 (BCM20702A0) - USB id 0a5c:21e8
+- Pluggable USB-BT4LE (BCM20702A1) - USB id 0a5c:21e8
+- SoundBot SB342 (BCM20702A1) - USB id 0a5c:21ec
+- StarTech USBBT2EDR4 (BCM20702A1) - USB id 0a5c:21e8
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- StarTech USBBT2EDR4 (BCM20702A0) - USB id 0a5c:21e8
```

File: source/_integrations/bluetooth.markdown [hunk 4]
```diff
@@ -102,6 +105,19 @@ Performance testing used the following hardware:
...
+### Known working adapters with missing firmware patch files
+
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
+- Cable Matters 604002-BLK (BCM20702A1) - USB id 0a5c:21e8
+- Kinivo BTD-400 (BCM20702A0) - USB id 0a5c:21e8
+- Pluggable USB-BT4LE (BCM20702A1) - USB id 0a5c:21e8
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- Pluggable USB-BT4LE (BCM20702A0) - USB id 0a5c:21e8
```

File: source/_integrations/bluetooth.markdown [hunk 5]
```diff
@@ -102,6 +105,19 @@ Performance testing used the following hardware:
...
 - External Adapters only: Home Assistant Blue running Home Assistant Operating System 9.3 with a USB extension cable.
 
+### Known working adapters with missing firmware patch files
+
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
+- Cable Matters 604002-BLK (BCM20702A1) - USB id 0a5c:21e8
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- Cable Matters 604002-BLK (BCM20702A0) - USB id 0a5c:21e8
```

File: source/_integrations/bluetooth.markdown [hunk 6]
```diff
@@ -102,6 +105,19 @@ Performance testing used the following hardware:
...
+
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
+- Cable Matters 604002-BLK (BCM20702A0) - USB id 0a5c:21e8
+- Kinivo BTD-400 (BCM20702A0) - USB id 0a5c:21e8
+- Pluggable USB-BT4LE (BCM20702A0) - USB id 0a5c:21e8
+- SoundBot SB342 (BCM20702A1) - USB id 0a5c:21ec
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- SoundBot SB342 (BCM20702A0) - USB id 0a5c:21ec
```

File: source/_integrations/bluetooth.markdown [hunk 7]
```diff
@@ -102,6 +105,19 @@ Performance testing used the following hardware:
 - Advertisements from an Oral-B iO Series 8
 - External Adapters only: Home Assistant Blue running Home Assistant Operating System 9.3 with a USB extension cable.
 
+### Known working adapters with missing firmware patch files
+
+These adapters do not currently have patch files with Home Assistant Operating System 9.3. The adapter may take multiple minutes to initialize after boot; eventually, they come online.
+
+While the Broadcom chips and drivers are generally performant, the experience may be sub-optimal until patch files are added.
+
+- ASUS USB-BT400 (BCM20702A1) - USB id 0b05:17cb
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- ASUS USB-BT400 (BCM20702A0) - USB id 0b05:17cb
```

File: source/_integrations/bluetooth.markdown [hunk 8]
```diff
@@ -79,18 +79,28 @@ If you experience an unreliable Bluetooth connection, installing a short USB ext
...
+- Techkey PBT06H (CSR8510A10)
+- TRENDnet TBW-107UB (CSR8510A10)
 
 📶 Denotes external antenna
 
 Performance is primarily determined by a combination of the chip and the Linux drivers for the adapter. Some vendors using the same chip had an unacceptable performance and are listed as unsupported.
 
 The following requirements must be met for an adapter to be labeled as High Performance:
 
+- Up by the time the Home Assistant Core container is started when using Home Assistant Operating System
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
```

File: source/_integrations/bluetooth.markdown [hunk 9]
```diff
@@ -102,10 +111,15 @@ Performance testing used the following hardware:
 - Advertisements from an Oral-B iO Series 8
 - External Adapters only: Home Assistant Blue running Home Assistant Operating System 9.3 with a USB extension cable.
 
+### Slow startup of Broadcom adapters on the ODROID N2+ platform
+
+The BCM20702 based adapters take an additional 120 seconds to initialize after boot with Home Assistant Operating System 9.3 when using an ODROID N2+; eventually, they come online.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
The BCM20702 based adapters may take an additional 120 seconds to initialize after boot with Home Assistant Operating System 9.3 when using an ODROID N2+; eventually, they come online.
```

File: source/_integrations/bluetooth.markdown [hunk 10]
```diff
@@ -102,10 +111,15 @@ Performance testing used the following hardware:
 - Advertisements from an Oral-B iO Series 8
 - External Adapters only: Home Assistant Blue running Home Assistant Operating System 9.3 with a USB extension cable.
 
+### Slow startup of Broadcom adapters on the ODROID N2+ platform
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
#### Slow startup of Broadcom adapters on the ODROID N2+ platform
```

**Review by frenck — APPROVED** [review]
Awesome, thanks @bdraco 👍

**Pull request merged**